### PR TITLE
Iss9: Basic HistoryStatistician

### PR DESCRIPTION
**Pull request by RiLeone**
Basic functionality of Statistician - we could merge it until more things to be done on this issue are defined. Also, I would first objectify states (#29) before extending this part.

### CODE REVIEW

File: src/HistoricStatistician.py [hunk 1]
```diff
@@ -7,12 +7,133 @@
...
+        self._res["summary"] = self.extract_summary()
+
+        return self._res
+
+
+    def print_results(self):
+        """Neatly print the results"""
+
+        keys_to_be_printed = ("battle_stats", "summary")
+        separator = "\n" + "=" * 80 + "\n"
```
Comment (mafilipp) [inline src/HistoricStatistician.py:46]: I would define it as a global constant to be used also in other funtions

File: src/HistoricStatistician.py [hunk 2]
```diff
@@ -7,12 +7,133 @@
...
+        for k2bp in keys_to_be_printed:
+            print("\n{:s}".format(k2bp.replace("_", " ").title()))
+            print("-" * len(k2bp))
+            pprint.pprint(self._res[k2bp])
+
+        print(separator)
+
+
+    def plot_world_population(self) -> pltlib.figure:
+        """Plot evolution of world population in the years."""
```
Comment (mafilipp) [inline src/HistoricStatistician.py:58]: I would add a legend: what are the red and green bars?

**Review by mafilipp — CHANGES_REQUESTED** [review]
Great Job!

I think that in the future we should add more figures to display the statistics.

**Comment by RiLeone** [comment]
Changes have been implemented. Merging in order to start to work on objectification of states.

**Pull request merged**